snapcraft.yaml

```diff
@@ -160,20 +157,24 @@ parts:
       - libmbim-proxy
       - libmbim-utils
       - libgudev-1.0-0
+      - libqrtr-glib0
 
     override-build: |
       set -ex
+      PREV="$(pwd)"
+      cd "$CRAFT_PART_SRC"
+      git tag --points-at HEAD
```

Comment:
This would just print the current tag to stdout, was this for debugging purposes?

Reply:
indeed, removed